### ISSUE TITLE
UCANs for app requests are missing their proof

### ISSUE THREAD

**Issue by icidasset** [fission-suite/webnative#104]
https://github.com/fission-suite/webnative/blob/a4347b31cfec987ff718882e29cc88cff696091e/src/apps.ts#L24

This line, and the other similar ones for the other requests, should be something along the lines of:
```js
proof: ucan_string
```
(see update-data-root function as an example: https://github.com/fission-suite/webnative/blob/a4347b31cfec987ff718882e29cc88cff696091e/src/data-root.ts)

## Getting the proof

This is the "difficult" part of the changes need to be made.
We need to get the actual UCAN to use as the `proof`.
The easiest way to get this is by doing:
```js
import * as ucanInternal from "./ucan/internal"
const proof = await ucanInternal.lookupFilesystemUcan("*")
```
We're revising various UCAN stuff soon, so this is just a temporary solution (hence the TODO's in the code).

PS. 
In case you were wondering, this workaround works because the auth lobby returns a UCAN with the `*` resource. That UCAN is stored in indexeddb, which we then lookup by using `lookupFilesystemUcan`.

**Comment by nathanmalishev** [comment]
Hey @icidasset, sorry for the delay in getting to this. Working on it now 👍 
Having a bit of trouble, i'm fetching the UCAN as suggested and then encoding it back as suggested [here](https://whitepaper.fission.codes/access-control/ucan/jwt-authentication), but the API is still yelling at me. (It doesn't actually return anything though to give me any hints)

I just wonder if i'm missing something in the expected flow that i have ? Any thoughts would be welcome!
```
const ucan = await ucanInternal.lookupFilesystemUcan("*")
  if (ucan === null) {
    throw "Could not find your local UCAN"
  }
  const prf = ucan.encode(ucan)

  const jwt = await ucan.build({
    audience: await api.did(),
    issuer: await did.ucan(),
    proof: prf, 
    potency: null
  })```

**Comment by icidasset** [comment]
@nathanmalishev I wouldn't use the whitepaper as a info source for writing code. A lot of that stuff we still need to implement 😉 But I agree that the server could give more information about what's missing/wrong.

Which status code are you getting from the server?
And can you show me what `jwt` results to?

**Comment by nathanmalishev** [comment]
@icidasset Well good news. It's working perfectly 🎆 
I believe with that flow, i believe there was some user error and i hadn't rolled up the sdk into my local test env 😆 .. Sorry about that.
Anyway the PR should be inc shortly :)

**Comment by icidasset** [comment]
Fixed in #111
